test/test_dependencies.py

```diff
@@ -189,3 +189,33 @@ def test_colcon_defaults(tmpdir):
     assert 'ros-dashing-ament-cmake' not in result
     assert 'ros-dashing-rclcpp' not in result
     assert 'ros-dashing-rclpy' in result
+
+
+@uses_docker
+def test_dummy_skip_rosdep_keys_doesnt_exist_pkg(tmpdir):
+    ws = Path(str(tmpdir))
+    pkg_xml = Path(ws) / 'src' / 'dummy' / 'package.xml'
+    pkg_xml.parent.mkdir(parents=True)
+    pkg_xml.write_text(CUSTOM_KEY_PKG_XML)
+    client = DockerClient()
+    platform = Platform(arch='aarch64', os_name='ubuntu', ros_distro='dashing')
+    skip_keys = 'definitely_does_not_exist'
```

Comment:
this test is incorrect, this line should be 

```
skip_keys = ['definitely_does_not_exist']
```

And the test should be that it _does not_ raise a `docker.errors.ContainerError`